src/test/java/net/sf/jabref/logic/formatter/OrdinalsToSuperscriptFormatterTest.java

```diff
@@ -2,29 +2,15 @@
 
 
 import net.sf.jabref.logic.formatter.bibtexfields.OrdinalsToSuperscriptFormatter;
-import org.junit.After;
 import org.junit.Assert;
-import org.junit.Before;
 import org.junit.Test;
 
+/**
+ * Tests in addition to the general tests from {@link net.sf.jabref.logic.formatter.FormatterTest}
+ */
 public class OrdinalsToSuperscriptFormatterTest {
-    private OrdinalsToSuperscriptFormatter formatter;
 
-    @Before
-    public void setUp() {
```

Comment:
Yes, for each test method a new instance of the test class is used. However, all the different instances are created before any test is run. So if the constructor of the formatter modifies some global static objects, then these changes might get lost.
Some discussion about this can be found here http://stackoverflow.com/questions/512184/best-practice-initialize-junit-class-fields-in-setup-or-at-declaration.
But in the end it is just a minor conventional thing...